src/quantuminspire/qiskit/quantum_inspire_provider.py

```diff
@@ -1,36 +1,37 @@
 from copy import copy
+from typing import List, Optional
 
 from coreapi.auth import BasicAuthentication
 from qiskit.providers import BaseProvider
 
+from quantuminspire.qiskit.backend_qx import QuantumInspireBackend
 from quantuminspire.api import QuantumInspireAPI
 from quantuminspire.exceptions import ApiError
-from quantuminspire.qiskit.backend_qx import QuantumInspireBackend
 
 QI_URL = 'https://api.quantum-inspire.com'
 
 
 class QuantumInspireProvider(BaseProvider):
     """ Provides a backend and an api for a single Quantum Inspire account. """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self._backends = []
         self._api = None
 
-    def __str__(self):
+    def __str__(self) -> str:
         return 'QI'
 
-    def backends(self, name=None, **kwargs):
+    def backends(self, name: str = None, **kwargs) -> List[QuantumInspireBackend]:
```

Comment:
again, should we not also typehint the `**kwargs`?

Reply:
comments says: **kwargs: Used for filtering, not implemented.
So it can be Any-thing...